### PR TITLE
Update Go version

### PR DESCRIPTION
**Pull request by mchompalova**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,24 +1,29 @@
...
+WORKDIR ${SRC_FOLDER}
+
+RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh \
+  && $GOPATH/bin/dep ensure -vendor-only \
   && mkdir -p $GOPATH/src/${ORG_PATH} \
   # Linking the project sources in the GOPATH folder
   && ln -s /${PROJECT}-sources $GOPATH/src/${REPO_PATH} \
-  && cd $GOPATH/src/${REPO_PATH} \
+  && cd ${SRC_FOLDER} \
   && echo "Fetching dependencies..." \
```
Comment (ivan-p-nikolov) [inline Dockerfile]: Could you move this comment to the before fetching dep, or remove it altogether?

File: Dockerfile [hunk 2]
```diff
@@ -1,24 +1,29 @@
...
-  && apk del .build-dependencies \
-  && rm -rf $GOPATH /var/cache/apk/*
+  && mv ${PROJECT} /${PROJECT}
+
+# Multi-stage build - copy certs and the binary into the image
+FROM scratch
 
 WORKDIR /
+COPY --from=0 /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=0 /${SRC_FOLDER} /
```
Comment (ivan-p-nikolov) [inline Dockerfile]: Shouldn't we just copy only the application? And not the source.

File: Dockerfile [hunk 3]
```diff
@@ -1,24 +1,29 @@
...
-  && cd $GOPATH/src/${REPO_PATH} \
+  && cd ${SRC_FOLDER} \
   && echo "Fetching dependencies..." \
-  && curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh \
-  && $GOPATH/bin/dep ensure -vendor-only \
   && go build  \
-  && mv ${PROJECT} /${PROJECT} \
-  && apk del .build-dependencies \
-  && rm -rf $GOPATH /var/cache/apk/*
+  && mv ${PROJECT} /${PROJECT}
```
Comment (ivan-p-nikolov) [inline Dockerfile]: We don't need this move.

**Review by ivan-p-nikolov — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (mchompalova) [reply Dockerfile]: You are right, I fixed it

File: Dockerfile [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
+
+# Multi-stage build - copy certs and the binary into the image
+FROM scratch
 
 WORKDIR /
+COPY --from=0 /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=0 /artifacts/* /
 
 EXPOSE 8080
 CMD [ "/coreos-version-checker" ]
```
Comment (MiroslavGatsanoga) [inline Dockerfile]: Newline :)

**Review by MiroslavGatsanoga — DISMISSED** [review]

**Review by ivan-p-nikolov — DISMISSED** [review]

**Review by ivan-p-nikolov — APPROVED** [review]

**Review by MiroslavGatsanoga — APPROVED** [review]

**Pull request merged**